src/main/java/com/skhu/gdgocteambuildingproject/projectgallery/model/GalleryProjectInfoMapper.java

```diff
@@ -25,4 +29,22 @@ public GalleryProjectInfoResponseDto map(GalleryProject project) {
                 .files(fileMapper.map(project.getFiles()))
                 .build();
     }
+
+    public GalleryProjectSummaryResponseDto mapToSummary(GalleryProject project) {
+        return GalleryProjectSummaryResponseDto.builder()
+                .galleryProjectId(project.getId())
+                .projectName(project.getProjectName())
+                .shortDescription(project.getShortDescription())
+                .serviceStatus(project.getServiceStatus().name())
+                .fileUrl(fileMapper.map(project.getFiles()).stream()
+                            .findFirst()
+                            .orElse(null))
```

Comment:
```suggestion
            .fileUrl(
                fileMapper.map(project.getFiles())
                          .stream()
                          .findFirst()
                          .orElse(null)
            )
```
이쪽 메서드 체인 들여쓰기 어떠신가영